### PR TITLE
Update of copy/paste support list

### PR DESCRIPTION
**Pull request by samy**
## Description
The copy/paste support list for Gutenberg has been updated since Gut can handle images copy/pasted from Word and Libre/Open Office

## How has this been tested?
Tested manually on:
- Word 2007/Windows/up-to-date WP 5.0
- Open/Libre Office on OS X and Ubuntu 

## Types of changes
Only readme update

## Checklist:
Only documentation update

### CODE REVIEW

**Comment by gziolo** [comment]
@mcsf and @iseulde - I don't know what the current status is but it would be great if you could confirm that.

@samy thanks for opening this PR and investing your time testing all those tools 💯

**Comment by iseulde** [comment]
Could you show in a quick video that pasting images from MS Word works? Last time I checked we were given a file path which is useless. The only thing we could do is adding an image placeholder.

**Review by mcsf — COMMENTED** [review]
Thanks for working on this, @samy! Since we're touching this table, I suggested a few additional things we can update.

File: packages/blocks/src/api/raw-handling/readme.md [hunk 1]
```diff
@@ -8,19 +8,20 @@ This folder contains all paste specific logic (filters, converters, normalisers.
...
 
 
 1. Apple Pages does not pass heading and image information.
-2. MS Word only provides a local file path, which cannot be accessed in JavaScript for security reasons.
-3. Still to do for MS Word Online.
-4. Evernote does not have headings.
-5. For caption and gallery shortcodes, see #2874.
+2. Still to do for MS Word Online.
+3. Evernote does not have headings.
+4. For caption and gallery shortcodes, see #2874.
```
[the remark this answers is not in the document]
Comment (iseulde) [reply packages/blocks/src/api/raw-handling/readme.md]: I don't remember why it was there. Perhaps for more info. Fine either way.

**Comment by iseulde** [comment]
@samy @mcsf Copy pasting a single image is not the same as copying the entire content. I don't think images are copied over when pasting the entire content? Perhaps a note that pasting single images works is good, but it still wouldn't work entirely.

**Comment by samy** [comment]
> @samy @mcsf Copy pasting a single image is not the same as copying the entire content. I don't think images are copied over when pasting the entire content? Perhaps a note that pasting single images works is good, but it still wouldn't work entirely.

I confirm pasting images + text doesn't work, it inserts image block + text blocks in Gut (it works only for single images)

**Comment by iseulde** [comment]
@samy I'm fine with updating the table, still saying images don't work, with a footnote that copying individual images works.

**Comment by iseulde** [comment]
For reference: https://core.trac.wordpress.org/ticket/46094 should fix this issue.

File: packages/blocks/src/api/raw-handling/readme.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mcsf) [reply packages/blocks/src/api/raw-handling/readme.md]: My suggestion is to remove and replace the cell content with `✓`.

**Comment by mcsf** [comment]
Hey @samy, are you up for applying the suggestions I attached when reviewing this PR? :)

**Comment by samy** [comment]
> Hey @samy, are you up for applying the suggestions I attached when reviewing this PR? :)

I resolved the conversation, so your changes have been included in my PR :) (if I understood how the PR/comments system works)

**Comment by iseulde** [comment]
Why does it have ✘ for MS Office heading? Where is the ✘ for images? Why is the comment about it removed? It's not a to do, it's impossible to do anything with local file paths. `[2]` also seems to be missing.

**Comment by samy** [comment]
> Why does it have ✘ for MS Office heading? Where is the ✘ for images? Why is the comment about it removed? It's not a to do, it's impossible to do anything with local file paths. `[2]` also seems to be missing.

Copying images from MS Office worked in Gut when I started this PR

**Comment by iseulde** [comment]
In https://github.com/WordPress/gutenberg/pull/13300#issuecomment-458465239 you confirmed that it doesn't work? It only works when you copy a single image.

**Comment by samy** [comment]
> In [#13300 (comment)](https://github.com/WordPress/gutenberg/pull/13300#issuecomment-458465239) you confirmed that it doesn't work? It only works when you copy a single image.

I will update my PR, sorry

**Comment by ellatrix** [comment]
@samy Something seems to have gone wrong pushing my changes to this branch. Sorry about that. I created a new branch and PR #15149 and merged the changes.

**Pull request closed without merging**